sklearn/metrics/ranking.py

```diff
@@ -149,14 +141,34 @@ def average_precision_score(y_true, y_score, average="macro",
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    interpolation : None (default), or string 'eleven_point'
```

Comment:
I think having strings is better because we might add others in the future, in particular the "correct" way ;)